### PR TITLE
[Doris on ES] Fix query failed when ES field value is null

### PR DESCRIPTION
**Pull request by e0c9**
## Proposed changes

when ES field value is null.  An exception occurs when parsing the field.
such as:
```
Scanner[0] process failed. status=Expected value of type: BIGINT; but found type: Null Type; Docuemnt source slice is : null 
```

## Types of changes

What types of changes does your code introduce to Doris?
_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [] New feature (non-breaking change which adds functionality)
- [] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [] Documentation Update (if none of the other choices apply)
- [] Code refactor (Modify the code structure, format the code, etc...)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [] I have created an issue on (Fix #ISSUE) and described the bug/feature there in detail
- [x] Compiling and unit tests pass locally with my changes
- [] I have added tests that prove my fix is effective or that my feature works
- [] If these changes need document changes, I have updated the document
- [] Any dependent changes have been merged

## Further comments

If this is a relatively large or complex change, kick off the discussion at dev@doris.apache.org by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by yangzhg — DISMISSED** [review]
LGTM

File: be/src/exec/es/es_scroll_parser.cpp [hunk 1]
```diff
@@ -319,6 +319,11 @@ Status ScrollParser::fill_tuple(const TupleDescriptor* tuple_desc, Tuple* tuple,
 
         void* slot = tuple->get_slot(slot_desc->tuple_offset());
         PrimitiveType type = slot_desc->type().type;
+
+        if (col.IsNull()) {
```
Comment (wuyunfeng) [inline be/src/exec/es/es_scroll_parser.cpp:323]: Plz add some comments for this

**Review by wuyunfeng — CHANGES_REQUESTED** [review]
I left some minor comment

File: be/src/exec/es/es_scroll_parser.cpp (hunk 1 above)
Comment (e0c9) [reply be/src/exec/es/es_scroll_parser.cpp:324]: done

**Review by yangzhg — APPROVED** [review]
LGTM

**Review by wuyunfeng — APPROVED** [review]

**Pull request merged**